numba/cuda/random.py

```diff
@@ -33,8 +33,16 @@
                                align=True)
 xoroshiro128p_type = from_dtype(xoroshiro128p_dtype)
 
+# When cudasim is enabled, Fake CUDA arrays are passed to some of the
+# @jit-decorated functions. This causes a fallback to object mode and the
+# emission of a warning at present - this is visible in the test suite with
+# cudasim, but also fallback-by-default will not be permitted in future. In
+# order to avoid the warning / future error, we explicitly specify that
+# object mode with loop lifting is acceptable when using the simulator.
+forceobj = looplift = config.ENABLE_CUDASIM
```

Comment:
```suggestion
_forceobj = _looplift = config.ENABLE_CUDASIM
```
perhaps to imply no-reuse/import from elsewhere?

Reply:
Good point - done.

Reply:
thanks